uiowa/uiowa
# Update file_mdm, sophron, imagemagick

Resolves https://github.com/uiowa/uiowa/issues/7108

# To Test

- Before pull this branch, on `main` sync down sandbox:

```
ddev composer install && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli media/add/image
```

- Use the images from the https://github.com/uiowa/uiowa/issues/7108 issue summary
- You should get an error after selection that prevents you moving forward with saving the media entity. Looking at the console log, you should see an error about base_convert() string/array type thing.

```
git checkout update_file_mdm && git pull && ddev composer install && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli media/add/image
```

Try the images again. There should be no error on upload and the images should appear in the correct orientation.

Review the original PR https://github.com/uiowa/uiowa/pull/264 and ImageMagick https://git.drupalcode.org/project/imagemagick/-/blob/8.x-3.x/src/Plugin/ImageToolkit/ImagemagickToolkit.php#L556 to understand that we aren't losing any functionality.

## Additional Notes

We still need these modules/enabled, but I don't think we are directly calling file_mdm in our code anymore.
